### ISSUE TITLE
factor out constants for option and var values

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#107]
Some option and var values that could be factored out as constants.  Cumulatively these might make a bit of a difference in memory footprint.  

More that 1 instance of these is created (number of instances in parens):

- [x] AxisOfSymmetryNode (2): font
- [x] CoefficientSlider (6): map, inverseMap, tickValues, trackSize, thumbSize
- [x] CoordinatesNode (14): font
- [x] GQCheckbox (19): font
- [x] GQGraphNode (4): otherCurves, decorations
- [x] GQManipulator (4): layoutCoordinates, phetioComponentOptions
- [x] PointToolNode (8): font
- [x] QuadraticNode (11): font

Only 1 instance of these is created:

- [x] DirectrixNode: font
- [x] ExploreInteractiveEquationNode.EquationNode: font
- [x] FocusAndDirectrixEquationNode: font, fractionFont
- [x] FocusAndDirectrixInteractiveEquationNode.EquationNode: font
- [ ] FocusAndDirectrixModel: numberPropertyOptions
- [x] GQScreenIconFactory: see defaults in createQuadraticNode
- [x] NoRealRoots: font
- [x] PointNode: layoutCoordinates
- [x] QuadraticTermsAccordionBox: font
- [x] StandardFormEquationNode: font
- [x] StandardFormInteractiveEquationNode: font
- [ ] StandardFormModel: numberPropertyOptions
- [x] VertexFormEquationNode: font
- [x] VertexFormInteractiveEquationNode: font
- [ ] VertexFormModel: numberPropertyOptions

**Comment by pixelzoom** [comment]
Before refactoring, startup heap size = 44.2MB, 2 minutes heap size = 51.5 MB. Chrome 70.0.3538.77 + macOS 10.11.6.

**Comment by pixelzoom** [comment]
Mentioning @samreid, since we had discussed the potential benefit of this on Slack, in relation to phetsims/tandem#71.

I tried factoring out constants for the default option values identified in https://github.com/phetsims/graphing-quadratics/issues/107#issue-379240254.  I did only the ones that are checked, since there are single instances of the other classes.  Here are the heap snapshots on Chrome 70.0.3538.77 + macOS 10.11.6, `?ea&brand=phet&fuzz`, before and after factoring out these constants.

Time (minutes) | Before | After |
--- | --- | ---
0 | 44.2 MB | 44.1 MB
2 | 51.5 MB |  51.7 MB 

Since there is no obvious memory reduction, I'm not going to commit the changes.
